### ISSUE TITLE
IOT Devices removed

### ISSUE THREAD

**Issue by n3roGit** [ioBroker/ioBroker.js-controller#775]
https://github.com/ioBroker/ioBroker.iot/issues/104

Hi guys,

I'm hoping someone can help here. For weeks I have had the problem that the devices of the IOT adapter disappear at irregular intervals. Sometimes just a few or even all of them.

The settings for iot disappear from the objects.

What I could reproduce now is that after activating influxdb on the objects the iot devices disappeared. I could not find any clues in the logs.
But before I used influxdb this already happened. I guess the iobroker "swallowed" some objects and destroyed the iot entries.

i really hope that someone can take care of this problem because i am really going crazy. i am also willing to give debugging access to my iobroker.

the system runs under docker on a ds918 and is installed in a current prod version.

node.js v10.19.0
NPM 6.13.4
js-controller 2.2.9

**Comment by Apollon77** [comment]
Which Admin version is installed?

**Comment by n3roGit** [comment]
The admin version is 3.7.8

**Comment by n3roGit** [comment]
any ideas?

**Comment by n3roGit** [comment]
Today, equipment disappeared again... Is there anything I can do to help solve the problem?

**Comment by Apollon77** [comment]
What you did before it disappeared? Normally nothing changes objects without an adapter or User triggered action.

**Comment by n3roGit** [comment]
I can definitely reproduce it if I activate the history on an object root.

**Comment by Apollon77** [comment]
What you mean with "object root"? history only can be activated on states normally ... Please use Admin 4.0.9 ... does it still happen there?

**Comment by n3roGit** [comment]
![image](https://user-images.githubusercontent.com/8565847/81052775-57715200-8ec4-11ea-87a3-fded65e40f6a.png)

Like This. Addong History for all my sonoff devices.

After doing this the IOT goes away.

I try it with the new admin version. give me a moment

**Comment by n3roGit** [comment]
Yes its the same. after addon History to all sonoff i have only 36 IOT devices and not 46
